spreadsheet/formula/functions_test.go

```diff
@@ -564,6 +564,29 @@ func TestMatch(t *testing.T) {
 	runTests(t, ctx, td)
 }
 
+func TestMax(t *testing.T) {
+
```

Comment:
remove extra newline at top of block

Reply:
Same in other Test functions below.

Reply:
See https://github.com/unidoc/unioffice/wiki/UniOffice-Developer-Guide#17-blank-lines